src/resources/views/auth/index.blade.php

```diff
@@ -0,0 +1,12 @@
+@extends('layouts.admin')
+
+@section('content')
+<h1>お客様へのお知らせ一覧画面</h1>
+@foreach ($infomation as $infomation1)
```

Comment:
[imo]
英語的に複数系がないのであれば、以下のような変数名の方が可読性が高いかと思います！
```suggestion
@foreach ($infomationList as $infomation)
```

Reply:
申し訳ございません。使わないようにします！